src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -1125,6 +1216,33 @@ private static bool TryGetTaskOfTType(TypeInfo taskTypeInfo, out TypeInfo taskOf
             bool IsTaskOfT(TypeInfo typeInfo) => typeInfo.IsGenericType && typeInfo.GetGenericTypeDefinition() == typeof(Task<>);
         }
 
+        private JsonRpcError CreateError(object id, Exception exception)
+        {
+            if (exception == null)
+            {
+                throw new ArgumentNullException(nameof(exception));
+            }
+
+            if (this.TraceSource.Switch.ShouldTrace(TraceEventType.Error))
+            {
+                this.TraceSource.TraceEvent(TraceEventType.Error, (int)TraceEvents.LocalInvocationError, "Exception thrown from request \"{0}\": {1}", id, exception);
```

Comment:
I think this error message might be more informative if it included the method name. Or, maybe that would be obvious from the stack trace?

Reply:
I agree. In an earlier iteration I had a method name in it, but since there are multiple places that errors are created, emitting the event from here ensured they were all captured -- but at the loss of the method name. 
The method name will typically show up in the stack trace, but if the original method itself is not async but returns a Task from an inner async method, it won't show up. So I'll circle back and look for a way to add it in all cases.

Reply:
Done.